src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+	<Project Include="System.Private.ServiceModel.csproj">
+      <OSGroup>Linux</OSGroup>
+	  <TargetGroup>dnxcore50</TargetGroup>
```

Comment:
Yes, I think we are ready.  It looks like Helix is already running our legacy ToF tests on at least Ubuntu, Centos, and OSX (and Windows CoreCLR).  So I'd say, yes, let's add entries to say we build for OSX and FreeBSD and see what Helix reveals.